packages/ui/src/components/molecules/inputs/DatePickerInput/DatePickerInput.tsx

```diff
@@ -80,7 +76,9 @@ export const DatePickerInput = ({
 
   const deserializeValue = useCallback(
     (value: DatePickerValue) => {
-      return dayjs(value, outputValueFormat);
+      const deserializedDate = dayjs(value, outputValueFormat);
+
+      return deserializedDate;
```

Comment:
Is there a reason to not shorten it?
```suggestion
      return dayjs(value, outputValueFormat);
```

Reply:
Sure, thing